### DISCUSSION TITLE
Handoff Issue

### DISCUSSION THREAD

**Discussion by Aspen-V** [ingrammicro/puzzle-publisher#89] [General]
I am not able to export with Handoff checked off. This happens only if I have complex symbol like with text style or layer style.

**Comment by Aspen-V** [comment]
It would be really great if we can export without handoff.

**Comment by MaxBazarov** [comment]
You can disable it in Configure Exporting.23 июня 2023 г., в 22:49, Aspen-V ***@***.***> написал(а):﻿
It would be really great if we can export without handoff.
